.gitignore

```diff
@@ -11,6 +11,10 @@
 !/bin
 /bin/*
 !/bin/brew
+!/share
+/share/*
+!/share/doc
+/share/doc/*
```

Comment:
I can confirm this observation. With these rules added, `git status` also starts showing untracked files in the documentation directory again.